lib/OpenSSL/RSATools.pm6

```diff
@@ -17,15 +17,17 @@ class OpenSSL::RSAKey {
 
     method new(:$private-pem, :$public-pem, :$x509-pem) {
         if $private-pem {
-            my $bio-buf = OpenSSL::Bio::BIO_new_mem_buf($private-pem.encode, $private-pem.encode.bytes);
+            my $private-pem-buf = $private-pem.encode;
```

Comment:
I think a comment would help to avoid someone "optimizing" away this variable.

Reply:
Done.